src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -1710,6 +1710,63 @@ public async Task<bool> IsAsyncMethod(SessionId sessionId, int methodId, Cancell
             return retDebuggerCmdReader.ReadByte() == 1 ; //token
         }
 
+        private bool IsClosureReferenceField (string fieldName)
+        {
+            // mcs is "$locvar"
+            // old mcs is "<>f__ref"
+            // csc is "CS$<>"
+            // roslyn is "<>8__"
+            return fieldName.StartsWith ("CS$<>", StringComparison.Ordinal) ||
+                        fieldName.StartsWith ("<>f__ref", StringComparison.Ordinal) ||
+                        fieldName.StartsWith ("$locvar", StringComparison.Ordinal) ||
+                        fieldName.StartsWith ("<>8__", StringComparison.Ordinal);
+        }
+
+        public async Task<JArray> GetHoistedLocalVariables(SessionId sessionId, int objectId, JArray asyncLocals, CancellationToken token)
+        {
+            JArray asyncLocalsFull = new JArray();
+            List<int> objectsAlreadyRead = new();
+            Regex regex = new Regex(@"\<([^)]*)\>", RegexOptions.Singleline);
```

Comment:
Maybe a field? IIUC, a single `MonoSDBHelper` instance is used for the full lifetime of `MonoProxy`.

Reply:
And name it to reflect what it is for.

Reply:
done